web/plugins/wato/check_crl_url.py

```diff
@@ -49,6 +49,14 @@ def _valuespec_active_checks_crl_url():
                     allow_empty = False
                 ),
             ),
+            (
+                'proxy',
+                HTTPUrl(
+                    title = _("Proxy"),
+                    help = _("Leave empty if none is neded. If a proxy is needed for the query it can be specified here. Will be uses as HTTP & HTTPS proxy."),
```

Comment:
Changed the help statement so that it is more clearly: [commit](https://github.com/jiuka/checkmk_crl_url/pull/2/commits/cc428325a9dbe38a7711a9327f38e067ef8f8abd)